src/server.js

```diff
@@ -51,6 +51,7 @@ function makeApp() {
 
 const app = makeApp();
 app.use(bodyParser.json());
+app.get(`${config.apiRoot}/applications/:id`, api.getApplicationStatus);
```

Comment:
Please take a look over at the PR I submitted about how to move the .get into the api.js where an Express router object is being returned  in general I like to create a single js file per API route (ie /application would be a single file and then /applications would be a 2nd js file).